src/reports/models/database/sequelize/sequelizeConnector.js

```diff
@@ -186,14 +187,18 @@ async function getReportsAndParse(query) {
     return allReports;
 }
 
-async function getLastReports(limit) {
-    const lastReports = getReportsAndParse({ limit, order: Sequelize.literal('start_time DESC') });
+async function getLastReports(limit, filter) {
+    const queryOptions = filter === 'is_favorite' ? { where: { is_favorite: true }, limit, order: Sequelize.literal('start_time DESC') } : { limit, order: Sequelize.literal('start_time DESC') };
```

Comment:
filter is generic and shouldn't be coupled to `is_favorite`

possible option to change to:

```suggestion
const queryOptions = { limit, order: Sequelize.literal('start_time DESC') };
    if (filter) {
        queryOptions.where = { };
        queryOptions.where[filter] = true;
    }
```

change also in `getReports`